packages/shared/src/components/fields/ProgressBar.tsx

```diff
@@ -41,7 +41,7 @@ export function ProgressBar({
           className?.bar,
           className?.barColor ?? 'bg-accent-cabbage-default',
         )}
-        style={{ width: `${percentage}%` }}
+        style={{ width: `${percentage >= 0 ? percentage : 0}%` }}
```

Comment:
Maybe we add `min-w-0 max-w-full` to handle those cases. Wdyt?

Reply:
negative width is invalid value in CSS